sem/HW/templates/authentication.html

```diff
@@ -5,22 +5,25 @@
 {% endblock %}
 
 {% block content %} 
-<h1>
-    Авторизация
-</h1>
+<h1>Регистрация</h1>
 
-<form action="/authentication/" method="post">
-    {% with messages = get_flashed_messages(with_categories=true) %}
-        {% if messages %}
-            {% for category, message in messages %}
-                <div class="alert alert-{{ category }}">
-                    {{ message }}
-                </div>
+<form method="post" action="{{ url_for('authentication_post')}}">
+    {{ form.csrf_token }}
+    {% for field in form if field.name != 'csrf_token' %}
+    <p>
+        {{ field.label }}<br>
+        {{ field }}
+        {% if field.errors %}
+        <ul class="alert alert-danger">
+            {% for error in field.errors %}
+            <li>{{ error }}</li>
             {% endfor %}
+        </ul>
         {% endif %}
-    {% endwith %}
-    <input type="text" name="username" placeholder="Введите ваше имя">
-    <input type="email" placeholder="Ваш email">
-    <input type="submit" value="Отправить">
+    </p>
+     {% endfor %}
+    <p>
+        <input type="submit" value="Register">
+    </p>
```

Comment:
То эту кнопку можно не создавать, так как она будет в объекте формы.
```suggestion
            <p>
	        {{ form.submit() }}
	    </p>
```
Хотя она должна появиться даже без этого